core/test/lib/tracehouse/trace-processor-test.js

```diff
@@ -634,6 +641,21 @@ Object {
           'largestContentfulPaintAllFramesEvt.ts': navigation.largestContentfulPaintAllFramesEvt.ts,
           'timestamps.firstContentfulPaintAllFrames': navigation.timestamps.firstContentfulPaintAllFrames, // eslint-disable-line max-len
           'timestamps.largestContentfulPaintAllFrames': navigation.timestamps.largestContentfulPaintAllFrames, // eslint-disable-line max-len
+
+          // In lcpAllFramesTrace (aka frame-metrics-m89) theres an FCP&LCP combo for the root frame and an inner frame.
+          // The root/outer frame (frame-metrics.html)'s paint is at 863ms.
+          // The inner frame (frame.html)'s paint is at 682ms
+
+          //     rootFrame: http://localhost:8080/frame-metrics.html
+          //     863ms since timeorigin
+          //     {"args":{"data":{"candidateIndex":1,"isMainFrame":true, ... ,"nodeId":8,"size":580,"type":"text"},"frame":"207613A6AD77B492759226780A40F6F4"},"cat":"loading,rail,devtools.timeline","name":"largestContentfulPaint::Candidate","ph":"R","pid":34523,"tid":775,"ts":23466886143},
+
+          //     innerFrame: http://localhost:8080/frame.html
+          //     682ms since timeorigin
```

Comment:
This test currently misidentifies the LCP-AllFrames. 

https://github.com/GoogleChrome/lighthouse/blob/74261f799ccdddba199701405eb282ad19fe8aed/core/test/lib/tracehouse/trace-processor-test.js#L674-L675

^  its just plain impossible for the allFrame variant to be smaller than the singleFrame variant.

Reply:
>  its just plain impossible for the allFrame variant to be smaller than the singleFrame variant.

Why is this impossible? Seems like this could happen if an iframe loads it's LCP faster than the outer frame. Our `perf-frame-metrics` smoke test was designed to do this.

https://github.com/GoogleChrome/lighthouse/blob/6909c6cc3a6b72b421fbedeabf06974734f7845d/cli/test/fixtures/perf/frame-metrics.html#L23-L25

Reply:
> its just plain impossible for the allFrame variant to be smaller than the singleFrame variant. 

This statement is true for FCP, but not LCP where the paint size is an additional criteria.  Example:

main/outer frame: 10sec and 2000px size
inner iframe:  5sec and 3000px size

LCPAF: 5s. 
LCP: 10s.